etc/tempesta_fw.conf

```diff
@@ -1129,3 +1129,39 @@
 # Default:
 #   Health monitor is disabled.
 #
+
+# TAG: client_db
+#
+# Path to a client database file used as a storage for clients info.
+# The same as cache_db.
+#
+# Default:
+#   client_db /opt/tempesta/db/client.tdb;
+#
+
+# TAG: client_tbl_size
+#
+# Size of client drop table.
+#
+# Syntax:
+#   client_tbl_size SIZE
+#
+# Default:
+#   client_tbl_size 16777216;  # 16MB
+#
+
+# TAG: client_lifetime
+#
+# Client life time in seconds. Life time of client accounting data after last
+# client connection was closed. The accounting data is used for Frang limits.
+# Zero value means unlimited life time.
+#
+# Syntax:
+#   client_lifetime NUM;
+#
+# Example:
+#   client_lifetime 3600;
+#
+# Default:
+#   client_lifetime 0;
+#
```

Comment:
We don't have any dependencies between this parameter and other *lifetime parameters.  The directive say `Life time of client accounting data after last client connection was closed`. It's closer to garbage collecting timeout: how many time to wait before _unused_ TfwClient entry will be destroyed. 

But dependencies between `session_lifetime` and frang limits truly exist.

> Probably it makes sense to generalize session_lifetime to client records lifetime and our sessions?
Sounds good. There was a discussion one day, that we would like to block certain clients, not proxies that they use. Probably frang accounting data should be also stored in `TfwHttpSess`?

Reply:
It could make sense in context of discussion https://github.com/tempesta-tech/tempesta/pull/1178/files#r255660386 . Previously we obtained `TfwClient` solely by IP address, i.e. we essentially limited TCP connections. The granularity didn't satisfy us and we came to obtaining clients based on HTTP headers, so now we have sticky cookie, User-Agent, X-Forwaded-For whatever and we should limit clients identified by their HTTP data.

And we can leave all TCP connection limiting to nftables layer while Tempesta cares solely on HTTP limiting - clear and simple.

Reply:
There is nothing about sticky cookies in #1115

Reply:
Please just add the lifetime descrption to the same Wiki page

Reply:
Decided that it still depends from the time frame http_resp_code_block